src/components/inspector/options-list.vue

```diff
@@ -332,27 +331,27 @@ export default {
     this.allowMultiSelect = this.options.allowMultiSelect;
   },
   methods: {
-    jsonDataChange() {	
-      let jsonList = [];	
-      try {	
-        jsonList = JSON.parse(this.jsonData);	
-        if (jsonList.constructor !== Array && jsonList.constructor !== Object) {	
-          throw Error('String does not represent a valid JSON');	
-        }	
-      }	
-      catch (err) {	
-        this.jsonError = err.message;	
-        return;	
+    jsonDataChange() {
+      let jsonList = [];
+      try {
+        jsonList = JSON.parse(this.jsonData);
+        if (jsonList.constructor !== Array && jsonList.constructor !== Object) {
+          throw Error('String does not represent a valid JSON');
```

Comment:
The try catch block is superfluous here. Inside of this if block why not just assign the error message to `this.jsonError` and return?

Reply:
I just removed the require statement that was throwing an error which caused the test to fail and the linter formatted the code. I can do the refactor as part of this PR?